guides/common/modules/proc_republishing-repository-and-content-view-metadata.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Republishing_Repository_and_Content_View-Metadata_{context}"]
+= Republishing Repository and Content View Metadata
+
+In the case of republishing repository metadata, this is used when a repository distribution does not have the content it should be distributing based on the contents of the repository.
+
+This procedure should be used with caution, and in most cases, a complete repository sync or publishing a new Content View version are the best options.
+
+.Procedure
+
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Select the product that includes the repository for which you want to republish metadata.
+Navigate to the repository where you wish to republish metadata.
+. To republish metadata for the repository, select *Republish Repository Metadata* from the *Select Action* menu.
++
+[NOTE]
+====
+This action is not available for repositories that implement the *Complete Mirroring* policy since the metadata is copied from the repository's upstream source rather than being generated locally.
```

Comment:
```suggestion
This action is not available for repositories that implement the *Complete Mirroring* policy since the metadata is copied from the repository's upstream source instead of being generated locally.
```